GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6_v2_with_addons.yaml

```diff
@@ -0,0 +1,23 @@
+subclass_name: composite.CompositeReader
+catalogs:
+  - catalog_name: dc2_object_run2.2i_dr6_v2pre
```

Comment:
This should be dc2_object_run2.2i_dr6_v2 ?

Reply:
yes, thanks for catching it